### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by KeeJef**
This will need to be checked so dont merge yet

### CODE REVIEW

File: src/i18n/fr-fr.js [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+        selectLocation: "SÉLECTIONNEZ L'EMPLACEMENT",
+        selectWalletFile: "SÉLECTIONNER LE FICHIER PORTEFEUILLE",
+        send: "ENVOYER",
+        sendCoins: "ENVOYER DES PIECES",
+        serviceNode: "NOEUD DE SERVICE",
+        settings: "RÉGLAGES",
+        showQRCode: "AFFICHER LE QR CODE",
+        showTxDetails: "AFFICHER LES DETAILS DE TRANSACTION",
+        stake: "STAKE",
+		sweepAll: "TOUT BALAYER",
```
Comment (Mikunj) [inline src/i18n/fr-fr.js]: Indentation needs fixing here

File: src/i18n/fr-fr.js [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+            title: "AFFICHER LES CLÉS PRIVÉES",
+            message: "Voulez-vous afficher vos clés privées ?",
+            ok: "AFFICHER"
+        },
+        stake: {
+            title: "Stake",
+            message: "Voulez-vous miser ?",
+            ok: "STAKE"
+        },
+		sweepAll: {
```
Comment (Mikunj) [inline src/i18n/fr-fr.js]: here too

File: src/i18n/index.js [hunk 3]
```diff
@@ -3,7 +3,8 @@ import enUS from "./en-us"
 // ADD LANGUAGES HERE
 const languages = [
     { name: "English", code: "en-us", flag: "us" },
-    { name: "Russian", code: "ru", flag: "ru" }
+    { name: "Russian", code: "ru", flag: "ru" },
+    { name: "French", code: "fr", flag: "fr" },
```
Comment (Mikunj) [inline src/i18n/index.js]:
```suggestion
    { name: "French", code: "fr", flag: "fr" }
```

File: src/plugins/timeago.js [hunk 4]
```diff
@@ -9,7 +9,8 @@ export default ({
         name: "Timeago",
         locale: "en",
         locales: {
-            ru: require("date-fns/locale/ru")
+        ru: require("date-fns/locale/ru"),
+	fr: require("date-fns/locale/fr"),
```
Comment (Mikunj) [inline src/plugins/timeago.js]:
```suggestion
	fr: require("date-fns/locale/fr")
```
Indentation needs to be fixed as well

**Comment by Mikunj** [comment]
`'strings.addressBookDetails'` is also missing in the translation

**Comment by KeeJef** [comment]
Is this ready to go? just resolved those merge conflicts in timeago and index.

**Pull request merged**